vizro-core/src/vizro/_vizro.py

```diff
@@ -88,19 +98,10 @@ def _reset():
         dash._pages.CONFIG.clear()
         dash._pages.CONFIG.__dict__.clear()
 
-
-def _append_styles(walk_dir: str, url_prefix: str) -> Tuple[List[Dict[str, str]], List[str]]:
-    """Append vizro css and js resources."""
-    _vizro_css = []
-    _vizro_js = []
-
-    for current_dir, _, files in sorted(os.walk(walk_dir)):
-        base = "" if current_dir == walk_dir else os.path.relpath(current_dir, walk_dir).replace("\\", "/")
-        for f in sorted(files):
-            path = os.path.join("/" + url_prefix, base, f) if base else os.path.join("/" + url_prefix, f)
-            extension = os.path.splitext(f)[1]
-            if extension == ".js":
-                _vizro_js.append({"src": path, "type": "module"})
-            elif extension == ".css":
-                _vizro_css.append(path)
-    return _vizro_js, _vizro_css
+    @staticmethod
+    def _get_external_assets(new_path: str, folder: Path, extension: str) -> List[str]:
```

Comment:
The output of this is identical to `_append_styles` but it uses pathlib, which we should generally prefer to `os.path`.